tests/creators/creator_stringclustering.py

```diff
@@ -0,0 +1,67 @@
+import datetime
+import sys
+sys.path.append("../..")
+
+
+def create():
+    from optimus import Optimus
+    from optimus.tests.creator import TestCreator, default_configs
+
+    op = Optimus("pandas")
+    df = op.create.dataframe({
```

Comment:
I think you should include at least one column with similar strings to properly test this functions.